tests/queries/tests.py

```diff
@@ -1172,6 +1172,16 @@ def test_lookup_constraint_fielderror(self):
         with self.assertRaisesMessage(FieldError, msg):
             Tag.objects.filter(unknown_field__name='generic')
 
+    def test_ticket_28297(self):
+        # https://code.djangoproject.com/ticket/28297
```

Comment:
Sure thing, I can add more descriptive comments here for sure. I think @charettes comment in the django ticket is correct though, I'd like to dig into this a bit deeper - there is something else at play here I think.